ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -530,3 +531,12 @@ func convertToULimits(cfgUlimits yaml.Ulimits) ([]*ecs.Ulimit, error) {
 
 	return ulimits, nil
 }
+
+// GoString returns deterministic string representation
+func SortedGoString(v interface{}) (string, error) {
```

Comment:
Thanks!  I think it'd be good to add a note that you're relying on this behavior.